### PR TITLE
2 new commands.

### PR DESCRIPTION
**Pull request by harshhh-dev**
Added !bam and !learnprogramming 

bam doesn't do anything. It just fake bans a user. 
learn programming links to the resources jlk always sends.

### CODE REVIEW

File: src/commands/bam.rs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+};
+
+#[command]
+#[aliases("kick", "ban")]
+#[description("Fake bans a user.")]
+#[usage("")]
+#[example = ""]
+#[help_available]
+#[bucket = "potentially_big_output_ever_channel"]
+pub(crate) async fn bam(ctx: &Context, msg: &Message) -> CommandResult {
```
Comment (lenscas) [inline src/commands/bam.rs:15]: I don't see an use for this one. It feels like all it does is trolling, which we don't want in the server.

**Review by lenscas — CHANGES_REQUESTED** [review]

File: src/commands/bam.rs (hunk 1 above)
Comment (harshhh-dev) [reply src/commands/bam.rs:15]: hm okay.

Comment (harshhh-dev) [reply src/commands/bam.rs]: deleted.

Comment (lenscas) [reply src/commands/bam.rs]: thanks :)

**Pull request merged**